presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/api/PackageAPI.java

```diff
@@ -129,13 +141,8 @@ public synchronized Response uninstall(
             @QueryParam("ignoreErrors") @DefaultValue("false") @ApiParam("If true, warnings are ignored during uninstall") boolean ignoreErrors)
     {
         LOGGER.debug("DELETE /package checkDependencies: {} ; ignoreErrors: {}", checkDependencies, ignoreErrors);
-        new Thread(new PrestoAsynchronousCommand(new PrestoUninstaller(installedPackageType, checkDependencies, ignoreErrors))).start();
-        return Response.status(ACCEPTED).entity(text("Presto is being uninstalled")).build();
-    }
-
-    public enum PackageType
-    {
-        RPM, TARBALL;
+        new Thread(new PrestoAsynchronousCommand(new PrestoUninstaller(RPM, checkDependencies, ignoreErrors))).start();
+        return Response.status(ACCEPTED).entity("Presto is being uninstalled. Please check back later using status api.").build();
```

Comment:
Yes. If the presto is not installed the status displays a message saying "presto is not installed"